articles/search/search-howto-index-cosmosdb.md

```diff
@@ -294,7 +294,7 @@ The generally available .NET SDK has full parity with the generally available RE
 
 ## Indexing changed documents
 
-The purpose of a data change detection policy is to efficiently identify changed data items. Currently, the only supported policy is the `High Water Mark` policy using the `_ts` (timestamp) property provided by Azure Cosmos DB, which is specified as follows:
+The purpose of a data change detection policy is to efficiently identify changed data items. Currently, the only supported policy is the [`HighWaterMarkChangeDetectionPolicy`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.search.models.highwatermarkchangedetectionpolicy?view=azure-dotnet) using the `_ts` (timestamp) property provided by Azure Cosmos DB, which is specified as follows:
```

Comment:
Please avoid including `en-us` in the links. Link should be `https://docs.microsoft.com/dotnet/api/microsoft.azure.search.models.highwatermarkchangedetectionpolicy`

Reply:
Fixed. 👍